src/js/package.json

```diff
@@ -24,7 +24,7 @@
     "@types/mocha": "7.0.2",
     "@types/node": "14.0.23",
     "fast-crc32c": "2.0.0",
-    "inotifywait": "1.8.3",
```

Comment:
I would roll this commit into the other. The reason is I think it makes sense for the path to build or be logically 'correct' at each commit. A reason to not do this would be to break up the patch to make things easier to read for reviewers, but this is a one line change, so not applicable here

Reply:
Got it

Reply:
It makes sense, I'll roll this.

Reply:
Done